### PR TITLE
Tidy up

### PR DESCRIPTION
**Pull request by adamnfish**
Fixes the CloudFormation template, which was not in sync with reality. The S3 prefix path now includes the stage. This CF template is currently deployed in CODE.

Switch to node 10.x (from 8.10) and add a `.nvmrc` file to pin it locally. The tests only ran on node 10 but the Lambda was using the node8 runtime in PROD. This unifies the AWS and dev environments on node 10. I've already updated the TeamCity build to use v10.

The TeamCity build now also runs the tests.

Removes code that was left over from the time when there was a local dev server. We may restore that functionality, but for now it isn't needed.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -71,11 +71,11 @@ const putConsentToFirehose =
...
               console.log('Error writing to kinesis stream', err, err.stack);
               serviceUnavailable('Could not save consent record', callback);
-            }  // an error occurred
-            else {
-              console.log('seccusfully added record to Kinesis stream');
-              ok(data.RecordId, callback);
-            }  // successful response, remove later.
+            } else {
+              console.log(
+                  'seccusfully added record to Kinesis stream', data.RecordId);
```
Comment (janua) [inline src/index.ts]: seccusfully

**Comment by janua** [comment]
:+1:

File: src/index.ts (hunk 1 above)
Comment (adamnfish) [reply src/index.ts]: Thanks!

Comment (adamnfish) [reply src/index.ts]:
```suggestion
                  'successfully added record to Kinesis stream', data.RecordId);
```

**Pull request merged**